CMakeLists.txt

```diff
@@ -911,7 +911,7 @@ if(SDL_ASSEMBLY)
 
     if(SDL_LSX)
       cmake_push_check_state()
-      list(APPEND CMAKE_REQUIRED_FLAGS "-mlsx")
+      set(CMAKE_REQUIRED_FLAGS "-mlsx")
       check_c_source_compiles("
```

Comment:
@madebr, is this right? It seems odd that these flags would replace the required flags rather than add to them.

Reply:
Oh, never mind, this is inside a check state block.

Reply:
Indeed, it was wrong befre because `CMAKE_REQUIRED_FLAGS` is a string, not a list.
It needed to be: `set(CMAKE_REQUIRED_FLAGS "${CMAKE_REQUIRED_FLAGS} -mlsx")`.

Some things are space limited, other things are `;`-limited. Long live CMake ;)